### ISSUE TITLE
Yume does not catch errors if we try to install a package that is not available

### ISSUE THREAD

**Issue by dikim33** [oscar-cluster/oscar#451]
**Reported by valleegr on 14 Jan 2008 23:19 UTC**
If you execute the following, no error is reported which creates a lot of issues to handle errors properly:

```
[oscar](root@centos-oscar2)# yume install toto
Setting up Install Process
Setting up repositories
oscar_rhel-5-x86_64       100% |=========================|  951 B    00:00
oscar_common-rpms         100% |=========================|  951 B    00:00
distro_centos-5-x86_64    100% |=========================|  951 B    00:00
Reading repository metadata in from local files
Parsing package install arguments
Nothing to do
[oscar](root@centos-oscar2)# echo $?
0
```

Yume should return an error is the package we want to install is not available.

**Comment by dikim33** [comment]
**Comment by valleegr on 14 Jan 2008 23:27 UTC**
Note that the problem comes from yum that acts the same way. I guess we should search for the package and only then try to install it.
For instance, the command ''yume /tftpboot/distro/centos-5-x86_64 --repoquery --nevra toto'' does not return anything which shows that no package is available.

**Comment by dikim33** [comment]
**Modified by valleegr on 14 Jan 2009 03:45 UTC**

**Comment by dikim33** [comment]
**Comment by valleegr on 14 Jan 2009 04:37 UTC**
(In [7839]) This check-in fixes #452 - Yume now returns an error code when we try to 
install a package that is not available.
